src/components/avatar-image.tsx

```diff
@@ -14,15 +13,15 @@ const ImageContainer = styled.div`
 `;
 
 interface AvatarImageProps {
-  src?: string | null;
+  email?: string | null;
 }
 
 export const AvatarImage: React.VoidFunctionComponent<AvatarImageProps> = ({
-  src,
+  email,
 }) => {
   return (
     <ImageContainer>
-      <StyledImg src={src || defaultAvatarUrl} />
+      <StyledImg email={email || ""} default="wavatar" rating="g" />
```

Comment:
```suggestion
      <StyledGravatar email={email || ""} default="mp" rating="g" />
```

Looking at the [docs](https://en.gravatar.com/site/implement/images/), `default=wavatar` may look to funny for our context.

![image](https://user-images.githubusercontent.com/608862/149672466-6b321e7d-2535-4c51-8942-ec67d358d42e.png)

`default=mp` looks neutral:

![image](https://user-images.githubusercontent.com/608862/149672497-38404506-bd78-4c74-9b5d-c4b7702d6079.png)